features/steps/files.py

```diff
@@ -38,11 +40,25 @@ def when_i_create_file_with_content(context, file_path, machine="uaclient"):
     text = context.text
     if "<ci-proxy-ip>" in text and "proxy" in context.instances:
         text = text.replace("<ci-proxy-ip>", context.instances["proxy"].ip)
+
+    date_match = re.search(r"<now(?P<offset>.*)>", text)
+    if date_match:
+        day_offset = date_match.group("offset")
+        offset = 0 if day_offset == "" else int(day_offset)
+        now = datetime.datetime.utcnow()
+        contract_expiry = now + datetime.timedelta(days=offset)
+        new_value = (
+            '\\"' + contract_expiry.strftime("%Y-%m-%dT00:00:00Z") + '\\"'
```

Comment:
```suggestion
            '"' + contract_expiry.strftime("%Y-%m-%dT00:00:00Z") + '"'
```
I think that will fix the tests

Reply:
Oh thanks, I was thinking the same, but wondering why it only affects xenial and focal lxd alone.
Thanks.